exp-player/addon/templates/components/exit-warning.hbs

```diff
@@ -0,0 +1,55 @@
+{{#if warn}}
+  {{#modal-dialog
+    targetAttachment="center"
+    translucentOverlay=true
+    container-class='exit-warning-modal'
+  }}
+  <div class="panel">
+    <div class="row">
+      <div class="col-xs-12">
+        <strong>Are you sure you want to leave the study early?</strong>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-xs-12 form">
+        <p>
+          Having difficulty with the study? Please leave us any comments below:
+        </p>
+        {{textarea value=exitReason class="exit-warning"}}
+        <hr>
+        <p>
+          If you do want to exit early, please select a privacy level for any video that was recorded during this session (<strong>*required</strong>):
+        </p>
+        <div class="form-group">
+          <div class="checkbox">
+            {{radio-button value="publicity and educatonal" checked=exitPrivacy}} Publicity and educational (video may be used for publicity or educa-onal purposes online or in the press in additon to for research purposes)
+          </div>
+          <div class="checkbox">
+            {{radio-button value="scientific" checked=exitPrivacy}} Scientific use only (e.g., a short clip may be shown at a scientific conference to demonstrate typical responses)
+          </div>
+          <div class="checkbox">
+            {{radio-button value="in-lab" checked=exitPrivacy}} In-lab use only (only researchers from MIT's Early Childhood Cogni-on Lab may view your video)
+          </div>
+          <div class="checkbox">
+            {{radio-button value="widthdraw" checked=exitPrivacy}}WITHDRAW your data from this session (your video will be destroyed and no one may view it or use any text you submitted
+          </div>
+        </div>
+      </div>
+    </div>
+    <hr>
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="btn-group pull-right">
+          <button class="btn btn-default exit-warning-btn" {{action 'onReject'}}>
+            Back
+          </button>
+          <button disabled={{if (not exitPrivacy) true false}} class="btn btn-warning exit-warning-btn" {{action 'onConfirm'}}>
```

Comment:
I think the truth helpers have to be inside of if/unless blocks

On 3/10/16 10:47 AM, Chris Seto wrote:
>
> In exp-player/addon/templates/components/exit-warning.hbs 
> <https://github.com/CenterForOpenScience/exp-addons/pull/48#discussion_r55698942>:
>
> > +            {{radio-button value="in-lab" checked=exitPrivacy}} In-lab use only (only researchers from MIT's Early Childhood Cogni-on Lab may view your video)
> > +          </div>
> > +          <div class="checkbox">
> > +            {{radio-button value="widthdraw" checked=exitPrivacy}}WITHDRAW your data from this session (your video will be destroyed and no one may view it or use any text you submitted
> > +          </div>
> > +        </div>
> > +      </div>
> > +    </div>
> > +    <hr>
> > +    <div class="row">
> > +      <div class="col-xs-12">
> > +        <div class="btn-group pull-right">
> > +          <button class="btn btn-default exit-warning-btn" {{action 'onReject'}}>
> > +            Back
> > +          </button>
> > +          <button disabled={{if (not exitPrivacy) true false}} class="btn btn-warning exit-warning-btn" {{action 'onConfirm'}}>
>
> Can't this just be |disabled={{not exitPrivacy}}|?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/CenterForOpenScience/exp-addons/pull/48/files#r55698942>.
>